include/field_player_avatar.h

```diff
@@ -3,7 +3,7 @@
 
 #include "save.h"
 #include "global.h"
-#include "field_system.h"
+#include "field_types_def.h"
```

Comment:
thank god (pikalax) for this name change tbh

Reply:
:3